BPMRFFEApp/src/Makefile

```diff
@@ -47,33 +24,12 @@ BPMRFFE_SRCS += BPMRFFE_registerRecordDeviceDriver.cpp
 BPMRFFE_SRCS_DEFAULT += BPMRFFEMain.cpp
 BPMRFFE_SRCS_vxWorks += -nil-
 
-# Link in the code from our support library
-BPMRFFE_LIBS += BPMRFFESupport
-
-# To build SNL programs, SNCSEQ must be defined
-# in the <top>/configure/RELEASE file
-ifneq ($(SNCSEQ),)
-    # Build sncExample into BPMRFFESupport
-    sncExample_SNCFLAGS += +r
-    BPMRFFE_DBD += sncExample.dbd
-    # A .stt sequence program is *not* pre-processed:
-    BPMRFFESupport_SRCS += sncExample.stt
-    BPMRFFESupport_LIBS += seq pv
-    BPMRFFE_LIBS += seq pv
-
-    # Build sncProgram as a standalone program
-    PROD_HOST += sncProgram
-    sncProgram_SNCFLAGS += +m
-    # A .st sequence program *is* pre-processed:
-    sncProgram_SRCS += sncProgram.st
-    sncProgram_LIBS += seq pv
-    sncProgram_LIBS += $(EPICS_BASE_HOST_LIBS)
-endif
-
 # Finally link IOC to the EPICS Base libraries
 BPMRFFE_LIBS += $(EPICS_BASE_IOC_LIBS)
+BPMRFFE_LIBS += asyn
+BPMRFFE_LIBS += calc
+BPMRFFE_LIBS += stream
```

Comment:
These libraries should be next to the DBD file block, because they should always be changed at the same time. Furthermore, this position makes it look like they are part of EPICS base, which they definitely aren't. Most importantly, and the reason I noticed this, they shouldn't be *after* epics base libs, they should always go before.

Reply:
Would you rather have them grouped by module as follows or keeping them grouped by dependency type (that is, simply having all libs together between DBDs and SRCS)?

```Makefile
# BPMRFFE.dbd will be created and installed
DBD += BPMRFFE.dbd

BPMRFFE_DBD += base.dbd

BPMRFFE_LIBS += asyn
BPMRFFE_DBD += asyn.dbd
BPMRFFE_DBD += drvAsynIPPort.dbd

BPMRFFE_LIBS += stream
BPMRFFE_DBD += stream.dbd

BPMRFFE_LIBS += autosave
BPMRFFE_DBD += asSupport.dbd

(...)

BPMRFFE_LIBS += $(EPICS_BASE_IOC_LIBS)
```

Reply:
I like grouping by module;